backend/substrapp/migrations/0017_alter_computetaskfailurereport_logs_and_more.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 4.2.3 on 2023-09-07 07:31
+
+import django.core.files.storage
+from django.db import migrations
+from django.db import models
+
+import substrapp.models.asset_failure_report
+import substrapp.models.function
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("substrapp", "0016_add_functionimage"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="computetaskfailurereport",
+            name="logs",
+            field=models.FileField(
+                max_length=36,
+                storage=django.core.files.storage.FileSystemStorage(),
+                upload_to=substrapp.models.asset_failure_report._upload_to,
+            ),
+        ),
+        migrations.AlterField(
+            model_name="functionimage",
+            name="file",
+            field=models.FileField(
+                max_length=500,
+                storage=django.core.files.storage.FileSystemStorage(),
+                upload_to=substrapp.models.function.upload_to_function,
+            ),
```

Comment:
@guilhem-barthes I would like to have your opinion but I'm not sure if this migration and the 0015 are not bogus migrations due to the fact that we are either using MinIO or the FileStorage depending if the deployment is local or not